scripts/airgapped/utils.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import os
+
+import docker
+
+cli = docker.client.from_env()
+
+LOG_FORMAT = "%(levelname)s \t| %(message)s"
+logging.basicConfig(format=LOG_FORMAT, level=logging.INFO)
+
+log = logging.getLogger(__name__)
+
+
+def delete_file_if_exists(file_name):
+    """Delete the file name if it exists."""
+    if os.path.exists(file_name):
+        try:
+            os.remove(file_name)
```

Comment:
The following one liner raises a `FileNotFoundError`:
```python
pathlib.Path("/tmp/testtt").unlink(missing_ok=False)
```
but looks like you'd rather `missing_ok=True`?

Reply:
done